### PR TITLE
Add operation to get tweets of specific handle

### PR DESCRIPTION
**Pull request by kapilraajP**
# Description
* Add new operation `getUserTweets` to get tweets in timeline with username/handle as parameter

**Test Configuration**:
* Ballerina Version: 2201.0.3
* Operating System: Ubuntu 20.04
* Java SDK: 11

# Checklist:

### Security checks
 - [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? 
 - [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: twitter/endpoint.bal [hunk 1]
```diff
@@ -403,6 +403,30 @@ public isolated client class  Client {
         return check handleStatusArrayResponse(httpResponse);
     }
 
+    # Get a user's tweets for a specific user.
+    # 
+    # + handleName - Handle name of user 
+    # + return - If success, returns 'Tweet' object array, else returns error
+    @display {label: "Get User's Tweets Of Specific User"} 
+    isolated remote function getUserTweets(@display {label: "Handle Name"} string handleName) 
+                                    returns @tainted @display {label: "Array Of Tweet"} Tweet[]|error {
```
Comment (LakshanSS) [inline twitter/endpoint.bal:412]: `@tainted` can be removed.
The use of @tainted is removed from Ballerina. It doesn't do anything

File: twitter/tests/test.bal [hunk 2]
```diff
@@ -126,6 +126,18 @@ function testGetUserTimeline() returns error? {
 @test:Config {
     dependsOn: [testGetUserTimeline], enable: true
 }
+function testGetUserTweets() returns error? {
+    log:printInfo("testGetUserTweets");
+    Tweet[] tweetResponse = check twitterClient->getUserTweets(screenName);
+    test:assertTrue(true, "Failed to call getUserTweets()");
```
Comment (LakshanSS) [inline twitter/tests/test.bal:132]: This line is not useful.
If we want to assert, we need to add some condition as the 1st parameter. But we are adding `true`. So always its gonna be true.

File: twitter/endpoint.bal (hunk 1 above)
Comment (kapilraajP) [reply twitter/endpoint.bal]: Removed

File: twitter/tests/test.bal (hunk 2 above)
Comment (kapilraajP) [reply twitter/tests/test.bal:132]: We can remove that line right?

Comment (kapilraajP) [reply twitter/tests/test.bal]: Ack. Updated

**Pull request merged**